blog/2025-01-14-build-3d-with-react.md

```diff
@@ -0,0 +1,97 @@
+
+    return <Entity>
+      {/* Create a camera entity */}
+      <Entity name='camera' position={[4, 3, 4]}>
+        <Camera clearColor='#111111' fov={28} />
+      </Entity>
+
+      {/* Create the spinning box entity */}
+      <Entity position={[0, 0.5, 0]}>
+        <Render type='box' material={material} />
+        <Script script={Spinner} speed={10}/>
+      </Entity>
+    </Entity>
+  };
+
+  ```
+
+</details>
+
+If you’re already familiar with the PlayCanvas API, you’re already off to a good start, you’ll see that any component system available in the engine can be used out of the box including rigid bodies, collision, physics etc. Follow our [Getting Started Guide](https://playcanvas-react.vercel.app/docs/guide/getting-started) and you'll be up to speed in no time.
+
+If you're coming from a React background, you'll find it' easy to integrate with existing React projects. Even if you’re new to PlayCanvas, you can pick up the basics in just a few hours.
+
+### Why React?
+
+React is loved by developers because it makes building user interfaces simple, intuitive, and predictable. With PlayCanvas React, you can now apply the same magic to 3D! Instead of writing complex, imperative code to create and manage 3D scenes, you can describe them declaratively, just like you do with HTML for a webpage. React’s component-based approach also makes it easy to reuse and organize your 3D content, so you can focus on creativity instead of boilerplate.
+
+### What about the Editor and Engine?
+
+- The PlayCanvas engine underpins everything. PlayCanvas React is simply a thin wrapper around the engine with a React friendly syntax. There's no strict rule for using one over the other, but if you're familiar with React and it's ecosystem, you'll find PlayCanvas React fits well with your stack.
+
+- The editor on the other hand is a powerful browser-based tool for building rich 3D content. PlayCanvas React is another way of achieving the same goal. Whilst the Editor provides a full suite of tools for asset management, scene organization, and rapid iteration, PlayCanvas React is more about building 3D content from within your React code. If you enjoy a code-centric workflow or want to integrate 3D content directly into a larger React application, PlayCanvas React may be the perfect fit.
+
+### Open Source FTW
+
+PlayCanvas React is completely open source, licensed under the [MIT License](https://github.com/playcanvas/react/blob/main/LICENSE). We believe that by sharing our work, collaborating with the community, and encouraging contributions, we aim to build a robust ecosystem.
+
+If you find a bug, have a suggestion, or want to contribute, feel free to [open an issue](https://github.com/playcanvas/react/issues) or submit a pull request. We welcome your feedback and ideas!
+
+### A new step for 3D
+
+The library is already being used in [production ready apps](https://ai.snapchat.com/) and we can’t wait to see what you build with PlayXanvas React!
```

Comment:
```suggestion
The library is already being used in production! [Snap AI]((https://ai.snapchat.com/)) from Snap Inc is built on PlayCanvas React:
```

Reply:
Please insert a little animated GIF of Snap AI's PlayCanvas React view here too.